cw-orch-interchain/examples/starship_ica_demo.rs

```diff
@@ -13,13 +10,13 @@ pub const STARGAZE: &str = "stargaze-1";
 pub const JUNO_FUNDS_DENOM: &str = "ujuno";
 
 fn main() {
```

Comment:
Have you ensures this test works? Should we add this test in CI?

Reply:
Yep, just checked, it runs. Do you think we want starship tests in CI?

Reply:
If possible yes, but it could be pain!

Reply:
i think the starship docs might have some snippet for it